styleguide/_themes/derek/scss/snowflakes/homepage.scss

```diff
@@ -330,7 +330,7 @@
 }
 
 .caseStudy-text {
-  font-family: $copy;
+  font-family: "Open Sans","Helvetica Neue",Helvetica,Arial,sans-serif;
```

Comment:
this got reverted to the wrong thing.  Can you make sure this is set to `$copy`?